samples-generator/lib/attribute.ts

```diff
@@ -0,0 +1,13 @@
+import { GltfType } from "./gltfType";
+
+export interface Attribute {
+    buffer: Buffer;
+    propertyName: string;
+    byteOffset?: number;
+    byteAlignment?: number;
+    componentType: GLenum;
+    count: number;
+    max: number[];
+    min: number[];
+    type: GltfType;
+}
```

Comment:
Some files have different line endings. This may not be a concern with TS, but just wanna be sure that you don't miss this, in case we need to keep them consistent.

Reply:
Are you talking about the trailing whitespace in each file?  All the files show up as `LF` line ending for me. Prettier automatically inserts a trailing newline, I don't automatically run prettier on every file though (because it would format parts of existing code that then show up as irrelevant changes in a pull request).

Reply:
I ran the formatter on everything new, but if you see anything else bad style-wise let me know.

Reply:
Your changes look good to me.